WcaOnRails/app/controllers/api/v0/user_roles_controller.rb

```diff
@@ -87,14 +101,17 @@ class Api::V0::UserRolesController < Api::V0::ApiController
 
     roles.concat(user.team_roles)
 
-    if user.admin? || user.board_member?
+    if user.board_member?
       roles << {
         group: {
-          id: user.admin? ? 'admin' : 'board',
-          name: user.admin? ? 'Admin Group' : 'Board Group',
-          group_type: UserGroup.group_types[:teams_committees],
+          id: 'board',
+          name: 'WCA Board of Directors',
```

Comment:
Actually here we are giving something that is in database. Since we have only english in database, I'm not changing here. Instead, now I have added the localization in Badges.jsx based on the group type. Hope that is fine.